### PR TITLE
Removes indexer benchmark binary, which is not an indexer benchmark.

### PR DESCRIPTION
**Pull request by nmarton-da**
changelog_begin
changelog_end

### CODE REVIEW

**Comment by skisel-da** [comment]
With this change the rest of //ledger/indexer-benchmark module becomes completely detached from sdk repo. I would then just move it completely out to oem kit repo.

File: ledger/indexer-benchmark/README.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
 # `indexer-benchmark`
 
-A tool that measures the performance of the indexer.
-
-# Input format
-
-This tool consumes kvutils ledger export files.
+A tool that measures the performance of the indexer. Used in downstream repositories.
```
Comment (skisel-da) [inline ledger/indexer-benchmark/README.md:3]: It cannot be used if it is removed from `artifacts.yaml`

**Comment by nmarton-da** [comment]
> With this change the rest of //ledger/indexer-benchmark module becomes completely detached from sdk repo. I would then just move it completely out to oem kit repo.

First we need to make a decision whether we want to support daml-sdk only indexer benchmark or not. Also then it needs to be clarified what will be the interface what we provide downstream, to build an indexer benchmark. (I agree it should be probably more limited than what it is now)
Please note: It was already detached also before this PR. The binary and the artifact it produces is not an indexer benchmark in the original sense, and is not usable in this form for benchmarking.
With this PR I merely intend to remedy a situation with removing the artifact and the binary so we are not accidentally using this as an indexer benchmark.

**Comment by skisel-da** [comment]
I mean with this change as it is defined now the code in  //ledger/indexer-benchmark becomes then completely orphan - it cannot be used from the outside, but it is still here.

File: ledger/indexer-benchmark/README.md (hunk 1 above)
Comment (nmarton-da) [reply ledger/indexer-benchmark/README.md:3]: Oh, right! And before we were even using a fat jar for a library dependency for the other repo.
Let me try to fix that.
Thanks for the headsup :+1:

**Review by skisel-da — APPROVED** [review]

**Pull request merged**